edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfigProvider.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Runtime.InteropServices;
+    using global::Docker.DotNet.Models;
+    using Microsoft.Azure.Devices.Edge.Agent.Core;
+    using Microsoft.Azure.Devices.Edge.Agent.Docker;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+
+    public class CombinedKubernetesConfigProvider : CombinedDockerConfigProvider
+    {
+        readonly IConfigSource configSource;
+
+        public CombinedKubernetesConfigProvider(IEnumerable<AuthConfig> authConfigs, IConfigSource configSource)
+            : base(authConfigs)
+        {
+            this.configSource = Preconditions.CheckNotNull(configSource, nameof(configSource));
+        }
+
+        static CreateContainerParameters CloneOrCreateParams(CreateContainerParameters createOptions) =>
```

Comment:
I'd like us to be more consistent with using qualifiers. Can we add them to the methods in this PR? I will make the same change in my PR.

Reply:
This is the applied style setting - private is not declared.